src/lib/storage/table.hpp

```diff
@@ -30,8 +30,11 @@ class Table : private Noncopyable {
   // We want a common interface for tables that contain data (TableType::Data) and tables that contain reference
   // segments (TableType::References). The attribute max_chunk_size is only used for data tables. If it is unset,
   // Chunk::DEFAULT_SIZE is used. It must not be set for reference tables.
-  explicit Table(const TableColumnDefinitions& column_definitions, const TableType type,
```

Comment:
Now seeing this, there are probably some other locations in our code where the `explicit` keyword flies around unnecessarily. Not sure if it's worth looking into though.

Reply:
Meh, doesn‘t hurt.